File: api-cmssignature/api-mssclient/mssclient/src/tr/gov/tubitak/uekae/esya/api/webservice/mssclient/wrapper/ProfileInfo.java
package tr.gov.tubitak.uekae.esya.api.webservice.mssclient.wrapper;

/**
 * Created with IntelliJ IDEA.
 * User: int2
 * Date: 19.07.2012
 * Time: 10:24
 * To change this template use File | Settings | File Templates.
 */
public class ProfileInfo {
    String issuerName;
    String serialNumber;
    String certIssuerDN;
    String mssProfileURI;
    String certHash;
    String digestAlg;

    public String getDigestAlg() {
		return digestAlg;
	}

	public void setDigestAlg(String digestAlg) {
		this.digestAlg = digestAlg;
	}

	public String getCertHash() {
		return certHash;
	}

	public void setCertHash(String certHash) {
		this.certHash = certHash;
	}

	public String getCertIssuerDN() {
        return certIssuerDN;
    }

    public void setCertIssuerDN(String certIssuerDN) {
        this.certIssuerDN = certIssuerDN;
    }

    public void setIssuerName(String issuerName) {
        this.issuerName = issuerName;
    }

    public void setSerialNumber(String serialNumber) {
        this.serialNumber = serialNumber;
    }

    public String getIssuerName() {

        return issuerName;
    }

    public String getSerialNumber() {
        return serialNumber;
    }


    public String getMssProfileURI() {
        return mssProfileURI;
    }

    public void setMssProfileURI(String mssProfileURI) {
        this.mssProfileURI = mssProfileURI;
    }

    public ProfileInfo() {

    }

    public ProfileInfo(String issuerName, String serialNumber) {

        this.issuerName = issuerName;
        this.serialNumber = serialNumber;
    }
}

File: api-cmssignature/api-mssclient/mssclient/csharp/src/wrapper/find/transaction/profile/ProfileInfo.cs
using System;

namespace tr.gov.tubitak.uekae.esya.api.webservice.mssclient.transaction.profile
{
    public class ProfileInfo
    {
        String _issuerName;
        String _serialNumber;
        String _certIssuerDn;
        String mssProfileURI;
        String certHash;
        String digestAlg;

        public String getDigestAlg()
        {
            return digestAlg;
        }

        public void setDigestAlg(String digestAlg)
        {
            this.digestAlg = digestAlg;
        }

        public String GetCertHash()
        {
            return certHash;
        }

        public void SetCertHash(String certHash)
        {
            this.certHash = certHash;
        }

        public String GetCertIssuerDN()
        {
            return _certIssuerDn;
        }

        public void SetCertIssuerDN(String certIssuerDN)
        {
            this._certIssuerDn = certIssuerDN;
        }

        public void SetIssuerName(String issuerName)
        {
            this._issuerName = issuerName;
        }

        public void SetSerialNumber(String serialNumber)
        {
            this._serialNumber = serialNumber;
        }

        public String GetIssuerName()
        {

            return _issuerName;
        }

        public String GetSerialNumber()
        {
            return _serialNumber;
        }

        public String getMssProfileURI()
        {
            return mssProfileURI;
        }

        public void setMssProfileURI(String mssProfileURI)
        {
            this.mssProfileURI = mssProfileURI;
        }

        public ProfileInfo()
        {

        }

        public ProfileInfo(String issuerName, String serialNumber)
        {

            this._issuerName = issuerName;
            this._serialNumber = serialNumber;
        }
    }
}
